ui/components/MesheryAdapterPlayComponent.js

```diff
@@ -406,8 +386,6 @@ class MesheryAdapterPlayComponent extends React.Component {
                     self.setState({ selectionError: false, cmEditorValAddError: false });
                   }
                 }}
-                onChange={(editor, data, value) => {
```

Comment:
Are you sure it doesn't exist to override any existing onChange handler of the component?

Reply:
It was but that part of code was already commented making this useless.

Reply:
Can you explain this a bit more, please?
I think the CodeMirror component is imported directly from `node_modules`. How was it's inbuild onChange handler commented?